examples/fit_fake_data.py

```diff
@@ -66,19 +78,29 @@
     text(
         0.6,
         0.5,
-        "LMFIT RESULT: %s" % newfit.latex(),
+        "lmfitdata RESULT: %s" % newfit.latex(),
         ha="center",
         va="center",
         color=thatline[0].get_color(),
         transform=ax.transAxes,
     )
+    text(0.6,0.5,(3*'\n')+list_symbs(newfit),
+            ha='center',va='top',
+            size=10,
+            color=thisline[0].get_color(),
+            transform = ax.transAxes)
```

Comment:
So I decided on a whim to add these back in, and I found that they were actually broken, not eliminated by choice!